src/modules/services_manager_service.rb

```diff
@@ -472,6 +483,114 @@ def refresh_services
       services.values.each(&:refresh)
     end
 
+    # Summary of services that have been started
+    #
+    # @return [String]
+    def started_services_summary
+      return "" unless started_services.any?
+
+      format(
+        _("The following services(%{amount}) will be started:<br />%{services}<br /><br />"),
```

Comment:
NP: I prefer the amount before the "services" word. It seems more natural to me.

Reply:
As commented offline, it usually appears after the word "foo(5)".